### PR TITLE
Relaunch: Blog

### PR DESCRIPTION
**Pull request by straight-shoota**
This patch implements the new design for the blog index and post pages.

### CODE REVIEW

File: _posts/2023-01-24-placeos-environments.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
-title: "Automating smart buildings with Crystal: how PlaceOS creates and manages cohesive environments"
+title: "Automating smart buildings with Crystal"
+subtitle: "How PlaceOS creates and manages cohesive environments"
```
Comment (beta-ziliani) [inline _posts/2023-01-24-placeos-environments.md:3]: 🙇

**Review by beta-ziliani — APPROVED** [review]
Looks great. Just one little fix and a couple of reminders for our future selves.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+/nix/store/fixkzy5asfwd6rfmfdniz3nxgik6wg8y-pre-commit-config.json
```
[the remark this answers is not in the document]
Comment (straight-shoota) [reply .pre-commit-config.yaml:1]: Oops, yeah that doesn't belong here.

File: _layouts/post.html [hunk 3]
```diff
@@ -3,61 +3,28 @@
...
-            </a>
-        </div>
-      </div>
+<article class="post" itemscope itemtype="https://schema.org/Article">
+  <hgroup>
+    <h1 class="post-title">{{ page.title }}</h1>
+    {% if page.subtitle %}
+    <h1 class="subtitle">{{ page.subtitle }}</h1>
+    {% endif %}
+  </hgroup>
```
[the remark this answers is not in the document]
Comment (straight-shoota) [reply _layouts/post.html:13]: What releases link?

File: _includes/posts/meta.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<time class="post-date" pubdate datetime="{{ include.post.date | date_to_xml }}"
+  >{{ include.post.date | date_to_string }}</time
+>
+{% assign authors = include.post.author | split: ',' %}
+<ul class="avatar-list" aria-hidden="true">
```
[the remark this answers is not in the document]
Comment (straight-shoota) [reply _includes/posts/meta.html:5]: Hm, I guess we'll have to do a design iteration on this then b/c I don't see a clear fix for this.

[the remark this answers is not in the document]
Comment (straight-shoota) [reply _includes/posts/meta.html:5]: Tracked in #423

File: _layouts/post.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (straight-shoota) [reply _layouts/post.html:13]: Tracked in https://github.com/crystal-lang/crystal-website/issues/423

File: _layouts/post.html [hunk 5]
```diff
@@ -3,61 +3,28 @@
...
-  <noscript>
-    Please enable JavaScript to view the
-    <a href='http://disqus.com/?ref_noscript'>comments powered by Disqus.</a>
-  </noscript>
-  <a class='dsq-brlink' href='http://disqus.com'>
-    comments powered by
-    <span class='logo-disqus'>Disqus</span>
-  </a>
-</main>
+  <footer>
```
[the remark this answers is not in the document]
Comment (straight-shoota) [reply _layouts/post.html:19]: Tracked in https://github.com/crystal-lang/crystal-website/issues/423

**Review by beta-ziliani — APPROVED** [review]

**Pull request merged**